src/components/Form/СreateUserAccountForm/styles/CreateUserAccountForm.style.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+export const useStyles = makeStyles(
+  (theme: Theme) => ({
+    FormBox: {
+      '& ': {
+        display: 'flex',
+        gap: '15px',
+      },
+      '& .MuiFormLabel-root': {
+        fontFamily: 'Raleway',
+        color: 'black',
+        fontWeight: 600,
+        fontSize: theme.spacing(3.2),
```

Comment:
I used, because saw we used it in some components. Its used for more consistency, scalability. If we change spacing in our theme we can change all fontSize across our components instead of going manually and changing px

Reply:
I see what you mean. Reusability is typically a good thing.

However, here I find this fragile. Normally, paddings and font-size are mutually independent - you may have small text with large gaps, you may have large text with smaller gaps. Making font-size depending on default padding value leads for us to be unsure whether changing it will not make some parts unreadable at all.

Also, it's inconsistent: we already have `fontSize` as `theme.spacing(3.6)`, `theme.spacing(2.8)`, `theme.spacing(6.8)`, and also we have hard-coded values of 11, 14, 17, 18, 19, 20 and even more options - all in pixels.
 
In long run, I would suggest to introduce extra theme values with semantic names. E.g. `theme.fontSize.normal` and `theme.fontSize.heading1` or alike.

But in short run, and taking into account existing code, I suggest to use hard-coded value in `px`

Reply:
Okay, will do fix px value